releasenotes/notes/upgrade-metadata-json-schema-46f034ada7443cf9.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    When uploading a program, ``parameters``, ``return_values`` and ``interim_results``
+    should be specified as JSON Schema.
```

Comment:
Isn't there a bit more than this? Like the signature of `upload_program()` has changed, and these need to be under `spec`